Common/Packets/SecurityTypesPacket .cs

```diff
@@ -1,11 +1,11 @@
 ﻿/*
```

Comment:
Just noticed an extra space in the file name -- might want to fix, since it's such a small PR 😄